addons/event/views/event_views.xml

```diff
@@ -58,8 +58,8 @@
                                 <div class="o_setting_right_pane">
                                     <label for="use_mail_schedule"/>
                                     <div class="row mt16" attrs="{'invisible': [('use_mail_schedule', '=', False)]}">
-                                        <div class="col-lg-9">
-                                            <field name="event_type_mail_ids">
+                                        <div class="col-12">
+                                            <field name="event_type_mail_ids" style="width: 100%;">
```

Comment:
@aab-odoo Any plan to review the scss for settings views? Or is this the work of another team I am not sure? :/ But here on top of having a mix of bootstrap / odoo structure for settings... and I cannot understand the logic :/ We have a col-lg-9 but all fields in settings views have width 50% so the field is displayed as if it was a col-lg-4.50000 ? :') Is there a reason? Are we suppose to add width: 100% ourself or ...? https://www.awesomescreenshot.com/image/4241553/4fad73e04d67266b557702fdce2b487e

Reply:
@qsm-odoo I honestly don't know who is responsible of this, but what I can tell you is that there is no plan from our side to do anything about that.

Reply:
Thanks @aab-odoo 
@fja-odoo Let's keep your hack for now (just rebase/retarget/adapt), I have sent a feedback to NGR